lib/main.dart

```diff
@@ -46,10 +47,12 @@ class _MyAppState extends State<MyApp> {
     quickActions.initialize((String shortcutType) {
       if (shortcutType == 'action_one') {
         _service.sendSms(number);
+      } else if (shortcutType == 'action_two') {
+        _service.call(number);
       } else if (shortcutType == 'action_three') {
         _service.takePhoto();
       } else {
-        _service.call(number);
```

Comment:
I think there is quick action for calling also.

So what you can do here is create one separate action for both calling and opening the video screen.

Reply:
I already made a separate action for that...please take a look

<img src="https://user-images.githubusercontent.com/66327336/114050294-c3fd8980-98a9-11eb-807a-f8351b1ae973.png" width="1000">

Please see the demo video link mentioned above....
It will help you to understand the feature...

Thanks @shagun25 @hiashutoshsingh

Reply:
Got it.
Github wasn't showing the next lines. Thanks for the screenshot.